packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -206,9 +247,9 @@ OktaUtil.getRequestContext().app.value.id
 OktaUtil.getRequestContext().target.clientId
 ```
 
-There is also additional information available about the client app, such as `label`.
+There's also additional information available about the client app, such as `label`.
```

Comment:
Acrolinx flags 'additional' (at least on prod docs. Consider:

There's more information available about the client app...

Reply:
done